python/ray/_private/services.py

```diff
@@ -1286,6 +1286,7 @@ def start_dashboard(require_dashboard,
             import ray.dashboard.optional_deps  # noqa: F401
         except ImportError:
             if require_dashboard:
+                logger.exception("dashboard dependency error")
```

Comment:
What's the context for adding this?  Does the ImportError on the line below not show up in some cases?

Reply:
If I don't log this, I can't see which package import error.